model-optimizer/extensions/back/ReverseInputChannels.py

```diff
@@ -50,10 +50,24 @@ class InsertReverseChannels(BackReplacementPattern):
     """
     enabled = False
 
+    @staticmethod
+    def get_index_for_api_2_0(node: Node, idx: int):
```

Comment:
imho. I don't like this function name, hard to understand from name what function do. Possibly something like get_fw_index would be better.

Reply:
Done.